common/error.cpp

```diff
@@ -72,6 +72,9 @@ static String errorToString(ErrorCode errorCode) {
 	case kUserCanceled:
 		return _s("User canceled");
 
```

Comment:
Minor nitpick - maybe go with "Canceled by user" instead?

Reply:
> Minor nitpick - maybe go with "Canceled by user" instead?
I agree, it would be better to phrase it that way. However, this message is not part of the changes of this PR.

Reply:
Uh, right, I totally missed that. Sorry for the noise ;)